src/components/Modal/tests/Modal.test.tsx

```diff
@@ -312,18 +312,20 @@ describe('<Modal>', () => {
   });
 
   describe('with app bridge', () => {
+    let AppBridgeModalCreate: jest.SpyInstance;
```

Comment:
The next few files are using jest API to spy on functions rather than overwrite the function with a spy